pages.zh/linux/ark.md

```diff
@@ -5,12 +5,12 @@
 
 - 将存档解压缩到当前目录：
 
-`ark --batch {{存档名}}`
+`ark --batch {{路径/到/存档名}}`
 
 - 改变解压缩目录：
 
-`ark --batch --destination {{解压缩目录路径}} {{存档名}}`
+`ark --batch --destination {{路径/到/目录}} {{路径/到/存档名}}`
 
 - 创建一个原本不存在的存档并向它添加文件：
 
-`ark --add-to {{存档名}} {{文件1}} {{文件2}}`
+`ark --add-to {{路径/到/存档名}} {{路径/到/文件1个 路径/到/文件2个 ...}}`
```

Comment:
```suggestion
`ark --add-to {{路径/到/存档名}} {{路径/到/文件1 路径/到/文件2 ...}}`
```

Reply:
I found similar occurrences of this on other pages too, I will update it in a new PR.

Reply:
I found similar occurrences of this on other pages too, I will update it in a new PR.